lib/active_shipping/carriers/ups.rb

```diff
@@ -376,10 +384,18 @@ def build_shipment_request(origin, destination, packages, options={})
             # Required element. The company whose account is responsible for the label(s).
             build_location_node(xml, 'Shipper', shipper, options)
 
-            if options[:saturday_delivery]
-              xml.ShipmentServiceOptions do
+            xml.ShipmentServiceOptions do
```

Comment:
If `saturday_delivery` and `label_method_code` are blank then we shouldn't include this node.